purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1285,6 +1276,11 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         }
     }
 
+    private fun cacheCustomerInfoAndSendToDelegateIfChanges(info: CustomerInfo) {
```

Comment:
Did you want to name this like this?


```suggestion
    private fun cacheCustomerInfoAndSendToDelegateIfChanged(info: CustomerInfo) {
```